src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -29,7 +37,7 @@
   }
 
   &.#{$prefix}-controlbar-visible {
-    bottom: 5em;
-    transition: bottom $animation-duration-short ease-in;
+    height: 70%;
```

Comment:
You are using a relative height here, so I was assuming that the distance from the control bar to the subtitle would change depending on the height of the player (because 70% of the player height converted to pixels changes with the player height). I tested that and observed that the distance is always the same though, as if you specified the position absolutely (like it was before in `em`). 
And like it often happens while discussing an issue, while writing this I just remembered that the percentage is taken from the player width, even for the height property. That explains why the offset from the control bar didn't change, no matter what height I set the player to :)